cppcache/integration-test/testExpiration.cpp

```diff
@@ -378,8 +395,9 @@ BEGIN_TEST(TEST_EXPIRATION)
     //////////////
     RegionAttributes attrs_18;
     // ettl = 0, eit = 0, rttl = 0, reit = 0
-    attrs_18 = setRegionAttributesTimeouts(std::chrono::seconds(6), std::chrono::seconds(3),
```

Comment:
Seems like a lot of formatting changes. Are you using clang-format?

Reply:
Yes I ran all the source I touched through clang-format.

Reply:
What version of clang-format?